tests/test_model_layers.py

```diff
@@ -490,6 +490,45 @@ def test_CommunicationMean(model: capellambse.MelodyModel) -> None:
     assert cmd in comm.allocated_exchange_items
 
 
+@pytest.mark.parametrize(
+    "fc_uuid,link_uuid,target_uuid",
+    [
+        pytest.param(
+            "d588e41f-ec4d-4fa9-ad6d-056868c66274",
+            "41e2ff3c-bc49-4eb1-ace9-66920b21179d",
+            "55b90f9a-c5af-47fc-9c1c-48090414d1f1",
+            id="OperationalProcess",
+        ),
+        pytest.param(
+            "dfc4341d-253a-4ae9-8a30-63a9d9faca39",
+            "7311e58e-9500-4c72-9620-aac32e4a1458",
+            "1a414995-f4cd-488c-8152-486e459fb9de",
+            id="FunctionalChain",
+        ),
+    ],
+)
+def test_FunctionalChainInvolvementLink_attributes(
+    model_5_2: capellambse.MelodyModel,
```

Comment:
IMO it would be better to add this to the 5.0 test model, instead of adding some cases to the 5.2 one. Keeping as much as possible in the same model will make it easier to migrate to a higher minimum supported version in the future, because there's only one model with actual useful content.

Reply:
Tbh: Whenever I test if capellambse works on a new Capella model (for e.g. 6.0) I migrate the TestModel from the latest version I used before. So all of the content from 5.0 is in 5.2 and ongoing. I don't know how much stuff is missing in the 5.0 TestModel, but I know that I (and probably we) don't want to many test models in this repo. Personally I'd like to only have one TestModel for the latest stable Capella version.